runtime/onert/backend/cpu/KernelGenerator.cc

```diff
@@ -1188,15 +1219,17 @@ void KernelGenerator::visit(const ir::operation::Mean &node)
 {
   const auto output_index{node.getOutputs().at(0)};
   const auto input_index{node.getInputs().at(ir::operation::Mean::INPUT)};
+  const auto axes_index{node.getInputs().at(ir::operation::Mean::AXES)};
 
-  const auto axes = node.param().axes;
+  const auto &axes = _ctx.at(axes_index);
+  const auto axes_vec = getReducerAxes(axes);
```

Comment:
I wonder what happen if axes is output of other operation.
Shouldn't we check `if (axes.isConstant())` to call this function?

Reply:
There is no handling dynamic axes now. I will add an assertion instead.